src/ripple/app/ledger/impl/LocalTxs.cpp

```diff
@@ -150,8 +150,9 @@ class LocalTxsImp
                 return true;
             if (view.txExists(txn.getID()))
                 return true;
-            auto const sle = cachedRead(view,
-                keylet::account(txn.getAccount()).key, ltACCOUNT_ROOT);
+
+            std::shared_ptr<SLE const> sle = view.read(Keylet{ltACCOUNT_ROOT,
```

Comment:
The `account` keylet properly constructs a `Keylet` with type `ltACCOUNT_ROOT` for you already. So you can simply do: `sle = view.read(keylet::account(txn.getAccount()))`.